lib/deppack.js

```diff
@@ -301,6 +301,7 @@ const buildModMap = () => {
 };
 
 const isJs = (fileList, path) => fileList.find(path).type === 'javascript';
+const isTest = (path) => path.indexOf('-test') !== -1;
```

Comment:
I found in the documentation of conventions: "Files named -test.<extension> are considered tests."
 
I don't see any use case where brunch need to explore npm dependencies in files not compiled / delivered to the browser. Also those packages are not installed in production environment, it could break the compilation.

But if you prefer, I'm ok with adding an `config.npm.ignored` pattern :)